eng/Versions.props

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <MajorVersion>3</MajorVersion>
     <MinorVersion>1</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <PatchVersion>2</PatchVersion>
     <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
     <!-- Enable to remove prerelease label. -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

Comment:
This may cause pain downstream if merged because https://github.com/dotnet/runtime/issues/639 is not fixed yet. /cc @nguerrera @wtgodbe

I believe the fix is pretty simple, just change `<ItemGroup Condition="'$(StabilizePackageVersion)' == 'true'">` to `<ItemGroup>` below. Since this branch won't enter "master-branch mode" ever again, I believe this is fine. I only should need to find something better in dotnet/runtime.

Could you give it a try in this PR? We should be able to confirm the packages don't get created via the PR validation logs.

Reply:
i will give this a try

Reply:
We should not make this false in the PR, btw.

Reply:
yes this was opened before ur email. I have edited this in all other open prs